Fix/doc 4319: Kill Shindig documentation (#198)
exoplatform/exo-documentation: docs/whatsnew.rst

:ref:`Chat contacts filtering does not consider accents <Use-search-bar>`

Chat contact filtering is not sensitive to accents anymore.

|image11|

:ref:`Login History can now be disabled by configuration <Configuration.loginHistory>`

With the new property ``exo.audit.login.enabled``, it is possible to enable or disable the login history.

:ref:`Remove uploaded files from chat <Upload-file-chat>`

Files uploaded in chat rooms can now be deleted as any other message.

.. _TechnicalNovelties:

========================================
Technical novelties in eXo Platform 5.3
========================================

As usual, eXo Platform continues to devote efforts to enhance security and performance.
In fact, in eXo Platform 5.3, new :ref:`parameters <ESClient>` has been introduced to configure max 
concurrent requests to the ElasticSearch server.

We also upgraded UI layer with latest versions of Vue.js and Vuetify frameworks.
We invite you to read the whole tutorial to develop portlets in eXo with Vuetify and Vue.js 
in  :ref:`this tutorial <PLFDevGuide.DevelopingApplications.DevelopingPortlet.Vuetify>`.

> With 5.3 version, eXo Platform opted to remove `Shindig <https://shindig.apache.org/>`__, 

Reviewer: ```suggestion
In version 5.3, eXo Platform decided to remove `Shindig <https://shindig.apache.org/>`__, 
```

the component used to support Gadgets.
In fact `Shindig <https://shindig.apache.org/>`__ is no more maintained and this leads to many issues such as:

- The dependance to specific versions of some important libraries namely Guava and which is hard to upgrade

> - SSRF security issues exposure

Reviewer 2: I don't think we should expose security issues, since it is still available in older versions (5.2, ...).

Author: Oh yes, you're right! thanks for the catch

Author: done
